test/test_distributed.py

```diff
@@ -1176,6 +1176,18 @@ def _test_DDP_2iter(
             # Shuffle the input so that DDP input is different
             input = input[torch.randperm(batch_size)]
 
+        # Test that saving and loading works
+        # TODO: It should be possible to save the entire model,
+        # but this doesn't work at the moment.  Update this test
+        # when it does work.
+        tmp_file = tempfile.TemporaryFile()
+        torch.save(model_DDP.state_dict(), tmp_file)
+        tmp_file.seek(0)
+        saved_state_dict = torch.load(tmp_file)
+        for k in model_DDP.state_dict():
+            self.assertEqual(model_DDP.state_dict()[k],
+                             saved_state_dict[k])
```

Comment:
This test uses a different serialization path than multiprocessing, so it's not testing your change at all

Reply:
Now that I've turned of hook serialization everywhere, this is a necessary test :>